git-link.el

```diff
@@ -549,6 +556,16 @@ return (FILENAME . REVISION) otherwise nil."
                                 (format "L%s-L%s" start end)
                               (format "L%s" start)))))))
 
+(defun git-link-gitiles (hostname dirname filename branch commit start end)
+  (format "https://%s/%s/+/%s/%s"
```

Comment:
Shouldn't this be called `git-link-googlesource` as that is what the format matches?

[The problem with support for JGit and Gitiles](https://github.com/sshaw/git-link/issues/100#issuecomment-1159289486) is they are local severs and the URL format is not standard like it is for Google Source, GitHub, etc... One can configure the path to be anything they want, right?

Reply:
Ah I see now - I believe this format only matches the Gitiles servers that are configured for google source. Sorry for the confusion :facepalm: 

Somehow I thought I had this figured out, but now I see that [Gitiles servers are configurable](https://gerrit.googlesource.com/gitiles/+/HEAD/Documentation/config.md#URLs).

I'll update this per your suggestion, and post a clarification in that thread as well :+1: